Response/DatatableQueryBuilder.php

```diff
@@ -633,7 +633,7 @@ private function getMetadata($entityName)
      */
     private function getTableName(ClassMetadata $metadata)
```

Comment:
renamed the property to `$entityShortName` to keep consistency with $entityName` property